backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -565,7 +565,7 @@ export class ExperimentService {
         // creating queries docs
         const promiseArray = [];
         let queriesDocToSave =
-          (queries &&
+          (queries[0] &&
```

Comment:
What if queries is not defined?

Reply:
It will store queries as [] blank array in the experiment json object